### PR TITLE
feat: use V2 cost/analyze API && edit minor types

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
New API is comming!!!!!!

스크린샷 2023-09-05 오후 6 40 43

### CODE REVIEW

**Review by skdud4659 — APPROVED** [review]
LGTM!

**Review by piggggggggy — APPROVED** [review]
LGTM!

File: apps/web/src/services/cost-explorer/cost-analysis/CostAnalysisPage.vue [hunk 1]
```diff
@@ -54,7 +55,7 @@ const getQueryWithKey = (queryItemKey: string): Partial<CostQuerySetModel> => (c
 
 let unregisterStoreWatch;
 const registerStoreWatch = (currentQuery) => {
-    unregisterStoreWatch = watch(() => costAnalysisPageStore.currentQuerySetOptions, (options: Partial<CostQuerySetOption>) => {
+    unregisterStoreWatch = watch(() => costAnalysisPageState, (options: Partial<CostQuerySetOption>) => {
```
Comment (WANZARGEN) [inline apps/web/src/services/cost-explorer/cost-analysis/CostAnalysisPage.vue:58]: Oh.. it's impressive that `() => costAnalysisPageState` works!

**Review by WANZARGEN — APPROVED** [review]
LGTM

**Pull request merged**